toolflow/vivado/arch/axi4mm/axi4mm.tcl

```diff
@@ -132,14 +132,14 @@ namespace eval arch {
     set bd_inst [current_bd_instance .]
 
     # bypass existing AXI4Lite slaves
-    set lite_ports [list]
-    set lites [get_bd_intf_pins -of_objects $inst -filter {MODE == Slave && CONFIG.PROTOCOL == AXI4LITE}]
```

Comment:
Is this required? Does the change still work for axi stream (QSFP plugin, ...)?

Reply:
We need to change this (otherwise non-wrapped AXI-Full-Slaves are ignored).
But you are correct, the current version will break with e.g. AXI-Stream. Will update this